redhat-developer/developers.redhat.com
# [DEVELOPER-5454]  composer env dependencies

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5454

### Verification Process

* Build a local development environment using the docker-compose.yml file, for example the 'drupal-dev' environment. Everything should function as it normally does
* Build the production environment using the docker-compose.yml file. If you create a terminal session within the built image, you should see that the /vendor directory does not contain any dev dependencies.
